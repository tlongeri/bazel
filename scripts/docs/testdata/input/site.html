<html devsite>
<head>
  <meta name="project_path" value="/_project.yaml">
  <meta name="book_path" value="/_book.yaml">
</head>
<body>

<h1 class="page-title">Command-Line Reference</h1>

See the <a href="/docs/build#specifying-build-targets">User's Guide</a> for the
target patterns syntax.
<a href="https://bazel.build/docs/build#specifying-build-targets">This</a> is the same link, btw.

Please ignore the <a href="relative/link">relative link</a> and external links such as <a href="https://github.com/bazelbuild/bazel">GitHub</a>.

<img alt="wow" src = "/images/great.png"/>
<img src="https://bazel.build/images/greater.jpg" alt=""/>

</body>
</html>
