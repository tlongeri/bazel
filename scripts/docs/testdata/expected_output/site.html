<html devsite>
<head>
  <meta name="project_path" value="/_project.yaml">
  <meta name="book_path" value="/versions/6.6.6/_book.yaml">
</head>
<body>

<h1 class="page-title">Command-Line Reference</h1>

See the <a href="/versions/6.6.6/docs/build#specifying-build-targets">User's Guide</a> for the
target patterns syntax.
<a href="https://bazel.build/versions/6.6.6/docs/build#specifying-build-targets">This</a> is the same link, btw.

Please ignore the <a href="relative/link">relative link</a> and external links such as <a href="https://github.com/bazelbuild/bazel">GitHub</a>.

<img alt="wow" src = "/versions/6.6.6/images/great.png"/>
<img src="https://bazel.build/versions/6.6.6/images/greater.jpg" alt=""/>

</body>
</html>
